pyrobosim/pyrobosim/gui/world_canvas.py

```diff
@@ -9,6 +9,7 @@
 from matplotlib.backends.backend_qtagg import FigureCanvasQTAgg
 from matplotlib.pyplot import Circle
 from matplotlib.transforms import Affine2D
+from matplotlib.artist import Artist
```

Comment:
are you using this import, by the way, or can it be removed?

Reply:
Yes, it's not being used. Missed it